packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -82,6 +88,40 @@
         "scopeName": "text.html.basic",
         "path": "./syntaxes/html.tmLanguage.json"
       }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "salesforce-lightning-explorer",
+          "title": "Lightning Explorer",
```

Comment:
The text displayed to the end user needs to be replaced by a label that is defined on a `package.nls.json` file, e.g. https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/package.nls.json
This applies to line 96, 105 and 121